### PR TITLE
move pid control gains to sdf xml

### PR DESCRIPTION
**Pull request by EliaTarasov**
This pr moves hardcoded gains for pid controller of each gimbal axis to xml sdf model file.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Cool! Did you also tune the default model?

**Comment by TSC21** [comment]
@hsu can you take a look at these changes?

**Review by TSC21 — APPROVED** [review]
At a first glance, it looks good! @EliaTarasov any chance you could provide us an update on the `typhoon_h480` SDF with the defaults, or as Lorenz asked, with a tuned ones if you got the chance of doing it?

**Comment by EliaTarasov** [comment]
Yes i can do it with defaults and also can try to make some tweaking for typhoon_h480.

**Comment by TSC21** [comment]
> Yes i can do it with defaults and also can try to make some tweaking for typhoon_h480.

That would be useful thank you!

File: src/gazebo_gimbal_controller_plugin.cpp [hunk 1]
```diff
@@ -28,7 +28,7 @@ GZ_REGISTER_MODEL_PLUGIN(GimbalControllerPlugin)
 GimbalControllerPlugin::GimbalControllerPlugin()
   :status("closed")
 {
-  /// TODO: make these gains part of sdf xml
+  /// defaults if sdf xml doesn't contain any pid gains
```
Comment (TSC21) [inline src/gazebo_gimbal_controller_plugin.cpp]: @EliaTarasov can you please set constants on the `gazebo_gimbal_controller_plugin.h` to set the PID values here? Similar to what is done with other constants? Like `kPIDPitchP`, `kPIDPitchI`, etc.?

Comment (EliaTarasov) [reply src/gazebo_gimbal_controller_plugin.cpp]: Yes, it's done

**Comment by TSC21** [comment]
LGTM! @EliaTarasov if you have time to tune the PID for the Typhoon model, please issue a new PR with the tuned gains. Thanks!

**Pull request merged**